wiki/History_of_osu!/2009/en.md

```diff
@@ -6,7 +6,7 @@ stub: true
 
 ## January
 
-osu! makes its way to the iPhone from a port to the iPhone by nuudles. The project was given its own [subforum](https://osu.ppy.sh/community/forums/47).
+osu! makes its way to the iPhone due to a port done by [nuudles](https://osu.ppy.sh/users/21312). The project was given its own [subforum](https://osu.ppy.sh/community/forums/47).[^iphone]
```

Comment:
For what it's worth, the iPhone subforum linked above is unavailable to non-admins.